freightliner_cascadia_2012_dot_80550/drivers.launch.py

```diff
@@ -161,7 +161,7 @@ def generate_launch_description():
         declare_vehicle_calibration_dir_arg,
         declare_vehicle_config_dir_arg,
         driver_shutdown_group,
-        dsrc_group,
+        v2x_ros_driver,
```

Comment:
Similarly, let's rename to `v2x_driver_group` to maintain consistency with other vehicle configs